wremnants/datasets/datasetDict2018_v9.py

```diff
@@ -0,0 +1,183 @@
+import copy
+from utilities import common
+
+lumicsv = f"{common.data_dir}/bylsoutput_2018.csv"
+lumijson = f"{common.data_dir}/Cert_314472-325175_13TeV_UL2018_Collisions18_HLT_IsoMu24_v_CustomJSON.txt"
+
+BR_TAUToMU = 0.1739
```

Comment:
These numbers are repeated in the datasetDict for 2016, can we move them to a common place instead? For example into utilities.common?

Reply:
Yes, that's a good idea. Can be done.